sdk/Sdk.Analyzers/DiagnosticDescriptors.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.Azure.Functions.Worker.Sdk.Analyzers
+{
+    internal class DiagnosticDescriptors
+    {
+        private static DiagnosticDescriptor Create(string id, string title,string messageFormat, string category, DiagnosticSeverity severity)
+        {
+            string helpLink = $"https://aka.ms/azfw-rules?ruleid={id}";
+            return new DiagnosticDescriptor(id, title, messageFormat, category, severity, isEnabledByDefault: true, helpLinkUri: helpLink);
+        }
+
+        public static DiagnosticDescriptor WebJobsAttributesAreNotSuppoted { get; }
+            = Create(id: "AZFW0001", title: "Invalid binding attributes", messageFormat: "The attribute '{0}' is a WebJobs attribute and not supported in the .NET woker.",
```

Comment:
Spelling on `woker`

Also, should this be put in terms of .NET isolated instead of just .NET worker?